ssl/statem/statem_dtls.c

```diff
@@ -770,7 +770,7 @@ static int dtls_get_reassembled_message(SSL *s, int *errtype, size_t *len)
      * Fragments must not span records.
      */
     if (frag_len > RECORD_LAYER_get_rrec_length(&s->rlayer)) {
-        al = SSL3_AD_ILLEGAL_PARAMETER;
+        al = SSL_AD_ILLEGAL_PARAMETER;
```

Comment:
Ditto with the TLS_ case mentioned above.
(I assume to standardize on a single spelling, but I didn't see it mentioned in the commit message.)